dataschema/annotations.py

```diff
@@ -507,6 +507,17 @@ def annotate_column(self, column: Schema.Column):
         column.clickhouse_annotation.type_name = self.type_name
 
 
+class ClickhouseNestedType(ColumnAnnotation):
+    """Override the default ClickHouse type for a nested column w/ argument."""
+
+    def __init__(self, nested_type_name: int):
+        self.nested_type_name = nested_type_name
+
+    def annotate_column(self, column: Schema.Column):
+        column.has_clickhouse_annotation = True
```

Comment:
can you add a check here that the type of the column is nested ?
also add a shortcut for this in the `schema_types.py`

Reply:
Done! Added the check, shortcut, and tests.